motion_control/controllers/speed_pid_controller/include/speed_pid_controller/SpeedPIDControllerParameters.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 COGIP Robotics association <cogip35@gmail.com>
+// This file is subject to the terms and conditions of the GNU Lesser
+// General Public License v2.1. See the file LICENSE in the top level
+// directory for more details.
+
+/// @ingroup    speed_pid_controller Speed PID controller parameters
+/// @{
+/// @file
+/// @brief      Speed PID controller parameters
+/// @author     Gilles DOFFE <g.doffe@gmail.com>
+
+#pragma once
+
+// Project includes
+#include "motion_control_common/ControllerParameters.hpp"
+#include "pid_ng/PID.hpp"
+
+namespace cogip {
+
+namespace motion_control {
+
+/// @brief Speed PID controller parameters
+class SpeedPIDControllerParameters: public ControllerParameters {
+    public:
+        /// @brief Constructor
+        /// @param pid  PID parameters
+        SpeedPIDControllerParameters(
+            cogip::pid::PID *pid = nullptr
+        ): pid_(pid) {};
+
+        /// @brief Get PID parameters
+        /// @return     PID parameters pointer
+        cogip::pid::PID *pid() { return pid_; };
```

Comment:
cant :disappointed: 

Because of the `compute` method will modify the object itself:
https://github.com/cogip/mcu-firmware/blob/95bbd7139ead626524e7050b9418191cf7fd7377/motion_control/controllers/speed_pid_controller/SpeedPIDController.cpp#L29

https://github.com/cogip/mcu-firmware/blob/95bbd7139ead626524e7050b9418191cf7fd7377/lib/pid_ng/PID.cpp#L13

Reply:
I mean the function should be `const`, not the return value.

Reply:
done